### PR TITLE
[Doc] Make adding a label to unlabeled columns in `DatagridConfigurable` more discoverable

### PR DESCRIPTION
**Pull request by slax57**
Improvements suggested by https://stackoverflow.com/a/77001801/3975522

### CODE REVIEW

File: docs/Datagrid.md [hunk 1]
```diff
@@ -900,6 +900,8 @@ const PostList = () => (
 );
 ```
 
+**Tip:** You may need to clear your local storage to reflect the changes, as react-admin saves the computed column names in the Store.
```
Comment (djhi) [inline docs/Datagrid.md:903]: Might be worth linking to store upgrade (versionning) doc

File: docs/SelectColumnsButton.md [hunk 2]
```diff
@@ -89,4 +89,27 @@ const BookList = () => {
...
+            <TextField source="year" />
+            <FieldWrapper label="Actions">
+                <EditButton />
+            </FieldWrapper>
+        </DatagridConfigurable>
+    </List>
+);
+```
+
+**Tip:** You may need to clear your local storage to reflect the changes, as react-admin saves the computed column names in the Store.
```
Comment (djhi) [inline docs/SelectColumnsButton.md:115]: Same

File: docs/Datagrid.md (hunk 1 above)
Comment (slax57) [reply docs/Datagrid.md:903]: I disagree, to me this tip is targeting the developer, not the end user

File: docs/SelectColumnsButton.md (hunk 2 above)
Comment (slax57) [reply docs/SelectColumnsButton.md:115]: same

File: docs/Datagrid.md (hunk 1 above)
Comment (slax57) [reply docs/Datagrid.md]: @fzaninotto shares your opinion, so I'll make the change.
He also noted that there already exists a wrapper component, so I used it instead in the code examples.

**Review by djhi — APPROVED** [review]

File: docs/Datagrid.md [hunk 3]
```diff
@@ -881,25 +881,26 @@ const PostList = () => (
...
+            <WrapperField label="Actions">
                 <EditButton />
-            </FieldWrapper>
+            </WrapperField>
         </DatagridConfigurable>
     </List>
 );
 ```
 
+**Tip:** You may need to clear your local storage to reflect the changes, as react-admin saves the computed column names in the Store. For the same reason, you users may need to log out and in again to see the changes. Alternatively, you can leverage [Store Invalidation](./Store.md#store-invalidation) to do it automatically.
```
Comment (adguernier) [inline docs/Datagrid.md:902]:
```suggestion
**Tip:** You may need to clear your local storage to reflect the changes, as react-admin saves the computed column names in the Store. For the same reason, your users may need to log out and in again to see the changes. Alternatively, you can leverage [Store Invalidation](./Store.md#store-invalidation) to do it automatically.
```

Comment (slax57) [reply docs/Datagrid.md:902]: woops, I'll fix it on `master` then

Comment (slax57) [reply docs/Datagrid.md:902]: Done https://github.com/marmelab/react-admin/commit/00053dc73dfb30bd3391af16a44b156776f480ed

**Pull request merged**